src/constants/sfx.asm

```diff
@@ -1,102 +1,109 @@
-;MUSIC_GLITCHY                                  equ $4C
-;MUSIC_SILENCE                                  equ $4D
-;MUSIC_SILENCE                                  equ $50
-;MUSIC_GLITCHY                                  equ $58
-;MUSIC_SILENCE                                  equ $59
-;MUSIC_GLITCHY                                  equ $5A
-;MUSIC_SILENCE                                  equ $5B
-;MUSIC_SILENCE                                  equ $5C
-;MUSIC_GLITCHY                                  equ $5D
-;MUSIC_SILENCE                                  equ $5E
-;MUSIC_SILENCE                                  equ $5F
-;MUSIC_SILENCE                                  equ $60
-MUSIC_COLOR_DUNGEON                             equ $61
-;MUSIC_UNTESTED                                 equ $6A
-;MUSIC_GLITCHED                                 equ $F0
-;MUSIC_NOTHING                                  equ $FF
+;MUSIC_SILENCE                                  equ $4A; More awakening...
+MUSIC_CATFISHS_MAW                              equ $4B
+;MUSIC_GLITCHY                                  equ $4C; Orchestral, short jingle.
+;MUSIC_SILENCE                                  equ $4D; harp jingle
+;MUSIC_MARIN_BEACH                              equ $4E
+;MUSIC_UNKNOWN                                  equ $4F; item jingle
+MUSIC_MINI_BOSS                                 equ $50
+;MUSIC_KANALET_CASTLE_COPY                      equ $51;	Copy of $13.
+;MUSIC_TAIL_CAVE_COPY                           equ $52;	Copy of $14.
+MUSIC_DREAM_SHRINE                              equ $53
+;MUSIC_UNKNOWN                                  equ $54; intense arpeggio, pre boss music? Switches to $19.
+MUSIC_FLYING_ROOSTER_REVIVED                    equ $55
+;MUSIC_UNKNOWN                                  equ $56; punch noises in upwards chord progression, plays $55 afterwards
+;MUSIC_CUCCO_HOUSE                              equ $57;	Is called 'Cucco House'
+MUSIC_FACE_SHRINE                               equ $58
+;MUSIC_WINDFISH                                 equ $59;	Is called 'The WindFish'
+MUSIC_TURTLE_ROCK                               equ $5A
+MUSIC_EAGLES_TOWER                              equ $5B
+MUSIC_BEFORE_EAGLES_TOWER_BOSS                  equ $5C;    Plays before the boss at the top of Eagle's Tower is summon by Grim Creeper.
+MUSIC_FINAL_BOSS_DIALOG                         equ $5D;	Plays during the final boss' dialog.
+MUSIC_BOSS_WARNING                              equ $5E;	Plays when the bosses tell you not to wake the windfish after defeating them.
+MUSIC_FINAL_BOSS_DEFEATED                       equ $5F
+MUSIC_ZELDA_NICKNAME_EASTER_EGG                 equ $60;	ZELDA nickname easter egg.
+MUSIC_COLOR_DUNGEON                             equ $61;
+;MUSIC_COLOR_DUNGEON_COPIES                     equ $62 to 6F;	Same as $61	
+;MUSIC_GLITCHED                                 equ $70 to $FE;	Glitches the music that played before it.
```

Comment:
This commented statement looks like a good way of signaling glitched music tracks 👍